components/transfer/TransferConfirmModal.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <NeoModal v-model="isModalActive" scroll="clip" @close="emit('close')">
+    <div
+      class="modal-width is-flex is-flex-direction-column is-justify-content-space-between">
+      <div>
+        <div
+          class="is-flex is-justify-content-center is-align-items-center header px-6 py-3">
+          <span class="is-size-5 has-text-weight-bold">
+            {{ $t('teleport.send') }}
+            <span class="is-uppercase">{{ unit }}</span>
+          </span>
+          <NeoIcon
+            class="position-right is-clickable mr-6"
+            icon="close"
+            pack="fas"
+            @click.native="closeModal" />
+        </div>
+        <div
+          class="px-6 is-scrollable"
+          :class="{
+            'is-bordered-top': isExpandList,
+          }">
+          <div
+            class="is-flex is-justify-content-space-between is-align-items-center py-4">
+            <span class="has-text-weight-bold is-size-6">{{
+              $t('transfers.network')
+            }}</span>
+            <div class="is-flex is-align-items-center">
+              <img class="mr-2 is-24x24" :src="tokenIcon" alt="token" />
+              {{ network }}
+            </div>
+          </div>
+          <div
+            class="is-flex is-justify-content-space-between is-align-items-center py-4 is-bordered-top">
+            <span class="has-text-weight-bold is-size-6 is-capitalized">{{
+              $t('general.from')
+            }}</span>
+            <div class="is-flex is-align-items-center">
+              <Avatar :value="accountId" :size="24" />
+              <span class="ml-2 is-size-6">
+                <Identity :address="accountId" hide-identity-popover />
+              </span>
+            </div>
+          </div>
+
+          <div
```

Comment:
You could have create a Table here instead of multiple div flex

Reply:
I have tried, but it would bring some extra stuff. I would keep the div implementation.

<img width="1143" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/29f95fe5-6cbc-4391-843f-8a4828dfb278">

Reply:
lol what